### ISSUE TITLE
No clear indication which tab is open

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1137]
See this screenshot. There's a little red line to indicate which pane is active, but this is too subtle to notice.
![whichtab](https://cloud.githubusercontent.com/assets/13840686/14035423/5cb8b668-f22e-11e5-8c20-f7bfa482cd01.jpg)

Much better would be to continue the white line over the inactive tabs like this:
![thistab](https://cloud.githubusercontent.com/assets/13840686/14035424/5dfcdae0-f22e-11e5-99ee-6dfa24bd99fe.jpg)

Or if you're attached to the full icons, like this:
![thistabwithicons](https://cloud.githubusercontent.com/assets/13840686/14035504/372b01ca-f22f-11e5-881c-74af312fc363.jpg)

**Comment by hwoarangmy** [comment]
Yes, it is not very easy to know the tab currently open. We need to mess with CEGUI to see if we can find a way to make it clearer.

**Comment by Bertram25** [comment]
Ok, this one is mine then. I wanted to add a ambient luminositybsetting first though. ;)
